admob/src/common/interstitial_ad_internal.cc

```diff
@@ -53,61 +50,6 @@ InterstitialAdInternal* InterstitialAdInternal::CreateInstance(
         // FIREBASE_PLATFORM_TVOS
 }
 
-void InterstitialAdInternal::SetFullScreenContentListener(
```

Comment:
Moved to a new internal full screen event listener base class which is shared with RewardedAds.